server/server.go

```diff
@@ -887,34 +885,35 @@ func (s *server) getClusterInfo() *rpcpb.ClusterInfo {
 	return info
 }
 
-func (s *server) handleUnexpectedNodeStop(unexpectedNodeStopCh chan network.UnexpectedNodeStopMsg) {
+func (s *server) handleUnhealthy() {
+	timer := time.NewTimer(0)
```

Comment:
Huh? Why wanting to drain the timer right away if you just created it? I don't think that will ever enter? 
Also, why not just create the timer with the `healthCheckFreq` right away? Creating with 0 doesn't seem to have any effect and it confuses me trying to understand what's going on...

Reply:
changed the logic here